package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@infura/sdk",
-  "version": "2.1.5",
+  "version": "2.2.5",
```

Comment:
The semantic versioning spec says it should go from `2.1.5` -> `2.2.0` because "Patch version MUST be reset to 0 when minor version is incremented":

https://semver.org/#spec-item-7